.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+	"singleQuote": true
```

Comment:
Spacing? Also add new line with every file, to keep it consistent with other files.

Reply:
Done with the changes